src/components/DataTable.tsx

```diff
@@ -386,6 +387,14 @@ export function DataTable<T extends object>(props: Props<T>) {
                           {t('delete')}
                         </DropdownElement>
                       )}
+
+                      {props.customActions &&
+                        props.customActions.map(
+                          (action: any, index: number) =>
```

Comment:
Can we try to typehint this? Code without `any` would be nice.

Reply:
I tried to make it as close as possible. Now, type is created. Let me know if you agree with this.